README.md

```diff
@@ -37,22 +37,22 @@ The IBM Cloud ibm_dpx_services Python SDK allows developers to programmatically
 
 Service Name | Module Name | Imported Class Name
 --- | --- | ---
-[DPX Service]() | data_product_exchange_api_service_v1 | DataProductExchangeApiServiceV1
+[Data Product Exchange Services](https://test.cloud.ibm.com/apidocs/dataproducts) | data_product_exchange_api_service_v1 | DataProductExchangeApiServiceV1
```

Comment:
```suggestion
[Data Product Exchange](https://cloud.ibm.com/apidocs/dataproducts) | data_product_exchange_api_service_v1 | DataProductExchangeApiServiceV1
```
if you follow my recommendations about the service name (i.e. add the x-alternate-name extension to your API definition), then the module name will be "dpx_v1" instead of "data_product_exchange_api_service_v1" and would be much easier for users to deal with.